cellxgene_schema_cli/setup.py

```diff
@@ -15,15 +15,6 @@
     install_requires=requirements,
     python_requires=">=3.10",
     packages=["cellxgene_schema"],
```

Comment:
i tried locally, and this definition still doesn't include validation_internals for me, since it doesn't look for nested directories. 

however, we can import "find_packages" from setuptools to find nested subdirectories without having to explicitly list them here

```suggestion
    packages=find_packages(exclude=["tests", "scripts"]),
```

Reply:
hm weird, okay i wonder why my local environment is different? i just pushed the change to use ` packages=find_packages(exclude=["tests", "scripts"]),` so hopefully that works for you